easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023b.yml

```diff
@@ -49,3 +49,6 @@ easyconfigs:
   - libspatialindex-1.9.3-GCCcore-13.2.0.eb:
       options:
         from-pr: 19922
+  - CFITSIO-4.3.1-GCCcore-13.2.0.eb:
+      options:
+        from-pr: 19840
```

Comment:
This doesn't make much sense anymore, since `CFITSIO/4.3.1-GCCcore-13.2.0` is already installed (via #479)

```suggestion
```